dags/taar_daily.py

```diff
@@ -167,15 +153,16 @@
         job_name="Train_the_Collaborative_Addon_Recommender",
         main_class="com.mozilla.telemetry.ml.AddonRecommender",
         jar_urls=[
+            # GCS bucket for testing is located in `cfr-personalization-experiment` project
+            # 'gs://taar_models/tmp/telemetry-batch-view-1.2.jar'
             "https://s3-us-west-2.amazonaws.com/net-mozaws-data-us-west-2-ops-ci-artifacts"
```

Comment:
should this change to a gcp bucket as well then? (instead of s3)

Reply:
good question. It requires reconfiguration of deployment pipeline of Scala jobs and I believe it is out of scope of the current task. It doesn't really affect TAAR operations, so we might think of this later. We can create an issue for this in `telemetry-batch-view` repo

Reply:
added note and issue